test/FateRewardControllerV3/MembershipReward_Withdraw.js

```diff
@@ -121,7 +130,7 @@ describe('FateRewardControllerV3.MembershipReward and Withdraw', () => {
             const receivedAmount = bobAfterLPAmount.sub(bobBeforeLPAmount)
 
             // check received amount after withdraw: withdrawAmount * (100 - lpWithdrawFee)
-            expect(receivedAmount).to.be.equal(withdrawAmount.mul(100 - 88).div(100))
```

Comment:
Use `.eq` on BigNumber objects to assert equality. This line is also causing the test to fail. Please debug it